AsyncDisplayKit/ASTableView.mm

```diff
@@ -675,6 +703,19 @@ - (void)tableView:(UITableView *)tableView didEndDisplayingCell:(_ASTableViewCel
 #pragma mark - 
 #pragma mark Batch Fetching
 
+- (void)_checkForBatchFetching
+{
+  // Dragging will be handled in scrollViewWillEndDragging:withVelocity:targetContentOffset:
+  if ([self isDragging] || [self isTracking] || ![self _shouldBatchFetch]) {
```

Comment:
Can we move this logic into ASBatchFetching.m, thus removing the duplication with the table and collection implementations? I know there are many other things that are duplicated in these files, which we can resolve with the plan to create a shared superclass, but that refactoring will be easier if it is clear that this logic is already unified.

Probably the best thing is to change the expected arguments of the ASBatchFetching.m call, to accept they scroll view instead of accessing the bounds, content size, and content offset. Because the bounds includes the content offset anyway, we should only request bounds and not call into the Apple framework to access content offset.

It would be fine to create a new function type to start splitting up some of the overall logic, but combining it into the single longer function that decides if it is the right time to batch fetch would be fine as well.